File: weak6.py/sentimental-mario-more/mario.py
# TODO

def get_height():
    while True:
        n = input("Height: ")
        if n > '0' and n < '9':
            return int(n)


Height = get_height()

for i in range(Height):
    j = Height - 1
    cont = 0
    while j >= 0:
        if j > i:
            print(" ", end="")
        else:
            print("#", end="")
            cont += 1
        j -= 1
    print(" ", "#" * cont)
    

File: weak1.c/marioV1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        printf("Height: ");
        scanf("%d",&height);
    }
    while (height>8 || height<1);
    int cont;
    for(int I=1;I<=height;I++)
    {
        cont=0;
        for(int J=1;J<=height;J++)
        {
            if(J>(height-I)){
                printf("#");
                cont++;
            }
            else
            {
                printf(" ");
            }

        }
        printf("  ");
        for(int J=0;J<cont;J++)
        {
            printf("#");
        }
        printf("\n");
    }
}